### ISSUE TITLE
Display Label for browse pages

### ISSUE THREAD

**Issue by alexdryden** [UIUCLibrary/Digital-Exhibits-Issues-Features#4]
Users report that the terms "Item" and "Item set" can be confusing public facing terms in the context of their collections. 

This might be a big lift from a development perspective, it might be worth seeing if there is a way to meet the need with existing options.

**Comment by alexdryden** [comment]
### Workaround:
Instead of making Browse pages from the navigation interface, make a standard page with Browse Preview from the Pages interface:
<img width="822" alt="Screenshot 2024-06-03 at 1 52 10 PM" src="https://github.com/UIUCLibrary/Digital-Exhibits-Issues-Features/assets/47127862/0163bb80-5c09-48ad-9a18-d4861f5d8ff2">

Public facing:
<img width="1455" alt="Screenshot 2024-06-03 at 1 55 31 PM" src="https://github.com/UIUCLibrary/Digital-Exhibits-Issues-Features/assets/47127862/7a81dead-5a58-43dc-9303-0a886e323e8d">


Use the preferred terminology for the page's url and the page title, and (optionally) the title for the Browse Preview block. Set the number of results to show to a large number so that all relevant results are displayed and set the Link Text to empty so that no "See all" link is displayed. 
 
### Evaluation:
This allows the correct terminology to be used in the title and the url slug, and it isn't significantly more work than creating a browse page. The major downside to this approach is that results will not be paginated, which may make this a poor fit if there are a lot of results. Also, leaving the "See all" link black is super hacky and could lead to confusing states (e.g., not all matching results are displayed in the "preview" but there is no indication that the result set is incomplete.)

Overall, I'd say this is a great fix and if this issue is a priority then a better solution needs to be developed.

**Comment by graykr** [comment]
@alexdryden That is an interesting hack with the high browse limit and blank "view more" text! It does seem to work.

**Comment by alexdryden** [comment]
@graykr if you think this will work for you, great! I'll keep this issue open in case we do find that the lack of pagination will be an issue.

**Comment by graykr** [comment]
@alexdryden I'd still like to see the original issue fixed, but it does seem like other issues would be more of a priority! We do still see the "items" label if we browse from the "metadata browse" link:

For example;
https://scp.library.illinois.edu/omeka-s/s/test-ihlc-oral-histories/item?Search=&property%5B0%5D%5Bproperty%5D=6&property%5B0%5D%5Btype%5D=eq&property%5B0%5D%5Btext%5D=Dana%20Miller

The other place it comes up (and perhaps should be a separate ticket) is with browsing item sets:
https://scp.library.illinois.edu/omeka-s/s/test-ihlc-oral-histories/item-set/9782

**Comment by alexdryden** [comment]
Yeah, those two end points, /item and /item-set, are going to be involved in a lot of pages and features, potentially. Making the change globally would be a too big of an undertaking for us to consider without upstream support. Because I don't think the developers would see the terminology as a bug, I'll start this discussion in the forum rather than their github repo, where I'm betting it wouldn't get much traction. I may not get to it this week, but if I get the discussion started, would you mind chiming in with some of the disciplinary reasoning for being able to control language used for those headings? That might encourage other non-technical users to chime in as well if they have similar needs.

**Comment by graykr** [comment]
@alexdryden That makes sense.  It would be wonderful if Omeka could do something with the "class" variable as an option to modify the label (so instead of "items" it could say "interviews" since the items are of class interview). I'm happy to add information as needed -- is the forum another login for me to create, or would this be adding detail to something you draft outside of the forum first?
